### DISCUSSION TITLE
GitHub Link gets redirect to https://github.com/ . Is it kept knowingly?

### DISCUSSION THREAD

**Discussion by AnirudhPanda** [npm/feedback#524] [Website]
Actually, the `npm` logo takes it to the Github organization.
Then what is the use of that Github Logo in the website exactly?

![tr](https://user-images.githubusercontent.com/66218496/129385417-9f4160e4-85ac-4c38-b4c8-c93e07de4a32.PNG)

**Comment by ljharb** [comment]
It's a link to npm's parent company.
